architecture/opp-architecture.adoc

```diff
@@ -5,6 +5,39 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-Discover {product-title} architecture and concepts.
+{product-title} is a single hybrid-cloud platform for enterprises to build, deploy, run, and manage intelligent applications securely across infrastructures. It is based on Red Hat Enterprise Linux, Kubernetes, and Red Hat {ocp}, and includes the following products:
 
-include::modules/opp-architecture-overview.adoc[leveloffset=+1]
\ No newline at end of file
+* Red Hat Advanced Cluster Management for Kubernetes - Controls clusters and applications from a single console.
+* Red Hat Advanced Cluster Security for Kubernetes - Provides information about cluster security, visibility management, and security compliance.
+* Red Hat Quay - A private container registry that stores, builds, and deploys container images.
+* Red Hat OpenShift Data Foundation Essentials - A data and storage services platform that provides a permanent place to store data while clusters spin up and down across environments.
+
+
+include::modules/opp-architecture-relnotes.adoc[leveloffset=+1]
+include::modules/opp-architecture-architecture.adoc[leveloffset=+1]
+include::modules/opp-architecture-compatibility-matrix.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../https://docs.openshift.com/container-platform/4.10/release_notes/ocp-4-10-release-notes.html[OpenShift Container Platform 4.10 release notes]
```

Comment:
These should be links, not xrefs. And, to keep users from bouncing between sites, please use access.redhat.com links instead of the docs.openshift ones.

Because these are all links, please incorporate them into the module.